### ISSUE TITLE
GA: recordEvent when `va-alert` loads on MHV landing page

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#68495]
### User story

As a Cartographers team member, I want to be able to segment GA data for users who see either the non-LOA3 or unregistered alerts, so that I can understand the frequency of this user type and see if their behavior differs significantly from other Veterans. 

### Notes

- "Key" user types include, at a minimum:
  - Those who are not registered at a facility (and would see corresponding alert)
  - Those whose credentials are not LOA3, and which credential they used (and would see corresponding alert)
- Component loaded event was added ~to~ **for use by the** `va-alert` component ~via~ by Analytics Team. [Implementation ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70772). Jamie Stutt "the custom event dataLayer" (her words) in a comment on that ticket
  - It's possible that instead of this va-alert event, we could segment users based on demographic data on the GA user (Carlos did some discovery around this topic in the past), but that is probably not necessary

### Example segmented GA report
<details><summary>Screenshot</summary>
<img width="501" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/059b3096-1747-4c67-9062-555d72c4091c">

</details>

## Acceptance criteria

- Built a GA report that shows unique link clicks segmented by non-LOA3 and unregistered user types
- ~GA events for link clicks include destination domain (VAgov vs eauth/myhealth.va.gov)~
- ~(stretch) Discovered how GA work might need to be adapted in GA4~
- ~Demonstrated that "key" landing page user types are can be used to segment/filter user sessions in Datadog~

**Comment by wesrowe** [comment]
planning: if a piece of work for GA segmenetation is identified as big-effort and wasted/duplicated between GA360 and GA4, we can consider ticketing as future work to be done when the need is clearly imminent.

**Comment by wesrowe** [comment]
I overhauled the ticket, reducing ACs to just implementing the component-load event on `va-alert` – expect the points should come down.

**Comment by wesrowe** [comment]
@radavis, we have partial success in Staging. Here's what I see for User 3, who is not registered:
<img width="514" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/46a236e0-b679-474b-8fb2-ef0b1e37f2b2">

**Comment by radavis** [comment]
@wesrowe Nice. I'm currently looking for a loa1 account, but none are listed in the Test User Dashboard.

**Comment by radavis** [comment]
Waiting on Platform review.

**Comment by radavis** [comment]
@wesrowe According to the [frontend support dashboard](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/), changes in department-of-veterans-affairs/vets-website#29421 are now in dev, staging, and prod. Let me know when you have the chance to verify in GA.

**Comment by wesrowe** [comment]
@radavis, have you tried it in Staging? I'm still seeing the same `Navigation - Unknown` in adswerve

**Comment by radavis** [comment]
@wesrowe Ok, we will need to sync w/ the Analytics Team to get this sorted.

**Comment by radavis** [comment]
@wesrowe Could you provide a link to the analytics.google.com page you are viewing?

**Comment by radavis** [comment]
On May 3rd, 2024, the Analytics Team was able to change the GTM settings so that AdSwerve outputs `Navigation - {heading content}` for `<va-alert />` components loading on our landing page.

There's still the problem of enabling this functionality for all `<va-alert />` components. [[discussion](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70772#issuecomment-2091586154)]

**Comment by wesrowe** [comment]
this looks perfecto on our "unregistered alert"!
<img width="472" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/3588574e-9838-4974-a803-371c14699714">

and the non-LOA3 alert too!
<img width="499" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/475c1499-2c49-40c1-9685-856fd01eab93">

**Comment by wesrowe** [comment]
@radavis, I believe I covered both use cases above. If you're satisfied, this can close.
